helm/charts/cytomine/templates/iam/iam-job-config.yaml

```diff
@@ -13,7 +13,7 @@ spec:
       - name: login
         image: "{{ .Values.images.iam }}"
         command: ["/bin/sh", "-c"]
-        args: ["/opt/keycloak/bin/kcadm.sh config credentials --server https://{{ .Values.global.domainName }}/iam --realm master --user admin --password $(KEYCLOAK_ADMIN_PASSWORD) --config /opt/keycloak-auth/kcadm.config"]
+        args: ["/opt/keycloak/bin/kcadm.sh config credentials --server http://iam.{{ .Release.Namespace }}.svc.cluster.local:{{ .Values.iam.port }}/iam --realm master --user admin --password $(KEYCLOAK_ADMIN_PASSWORD) --config /opt/keycloak-auth/kcadm.config"]
```

Comment:
Is this better to define `http://iam.{{ .Release.Namespace }}.svc.cluster.local:{{ .Values.iam.port }}/iam` once in `_helpers.tpl` and substitute it here?